### ISSUE TITLE
Playlist vanishing or "minimising" to top (Firefox)

### ISSUE THREAD

**Issue by ZacharyAmaranth** [YePpHa/YouTubeCenter#1767]
I've noticed in the last couple dev version updates, the playlist seems to change location some, either appearing side-by-side with the player or below it.  Now it's taken to vanishing, or just being a tiny block near the top of the screen.  It's erratic, and seems to persist when I disable other add-ons.  It also does this when I revert the player to "small."

I took screencaps of the two issues in question.  Refreshing sometimes changes the appearance, but usually not.

![playlist1](https://cloud.githubusercontent.com/assets/6621368/7548701/c4bc86d8-f5e3-11e4-92b7-57b532cff4e1.jpg)
![playlist2](https://cloud.githubusercontent.com/assets/6621368/7548702/c4bdf4a0-f5e3-11e4-809d-4e59fb01a8ed.jpg)

**Comment by Zarggg** [comment]
I'm seeing this as well, particularly the second example.

**Comment by Sollace** [comment]
Yup, I've had this as well. It used to fly down the page and get stuck there. At least it looks like they're working on it. It seems to be fixed when the player is enlarged but will still slide off the top of the page when the player is sized down.

**Comment by Sollace** [comment]
This should fix it:

    .watch-non-stage-mode #player-playlist {
      margin-top: 0px !important;
    }

**Comment by Sollace** [comment]
Playlist fix v2:

    .watch-non-stage-mode #player-playlist {
      position: initial !important;
      margin-top: 0px !important;
    }

This now more reliably puts it in the correct place.
